src/routes/docs/tutorials/react/step-6/+page.markdoc

```diff
@@ -5,7 +5,7 @@ description: Add a database to your React application using Appwrite Web SDK.
 step: 6
 ---
 # Create collection {% #create-collection %}
-In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store our ideas.
+In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store our ideas.  Adding user's permissions allows authorized users to view, create and delete documents.
```

Comment:
Please make sure to only have 1 space after periods.

> Adding user's permissions allows authorized users to view, create and delete documents.

I wouldn't know what exactly to do for permissions for this.

And, as a reminder, please make sure to use oxford commas.